src/main/java/core/basesyntax/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public Integer executeOperation(FruitDao fruitDao, FruitTransaction fruitTransaction) {
+        return fruitDao.putFruits(fruitTransaction.getFruit(), fruitTransaction.getQuantity());
+    }
```

Comment:
The `ReturnOperation` class should not directly interact with `FruitDao`. Instead, consider using the Strategy pattern effectively where operations are chosen based on the operation type. The logic to interact with the storage should be present in the service layer, and the strategy should only define the operation that needs to be performed.